### DISCUSSION TITLE
How to configure misfire instructions properly?

### DISCUSSION THREAD

**Discussion by jus0159** [quartznet/quartznet#2035] [Q&A] [answered]
I have a job which runs everyday Mon-Fri at 07:00 (Cron Trigger). The server is restarted once a week at Saturday 03:00.
I have tried two misfire configurations with following effects:
- SmartPolicy: Missed jobs are triggered, even these which are not really "missed" i.e. the job is triggered immediately after server restart on Sat 3:00 (not as expected)
- DoNothing: Missed jobs are ignored, even if the server was down on Mon between 6:00 and 8:00 then the missed job is not triggered (not as expected)

How to configure the trigger so that:
- the jobs are **NOT triggered** after downtime which **DOES NOT include** scheduled job
- the jobs are **triggered** after downtime which **includes** scheduled job

Thank you for your ideas.
Kind regards
Justyna

**Comment by lahma** [comment, marked as answer]
You need database persistence so that Quartz will know what hasn't been done after restart.
